package.json

```diff
@@ -44,8 +44,10 @@
         "@babel/preset-react": "^7.0.0",
         "@dhis2/app-runtime": "1.1.0",
         "@dhis2/cli-style": "3.1.1",
+        "@dhis2/d2-i18n": "1.0.4",
         "@dhis2/d2-i18n-extract": "1.0.7",
         "@dhis2/d2-i18n-generate": "1.1.1",
+        "@dhis2/ui-core": "2.0.2",
```

Comment:
Yarn doesn't install peer dependencies, so we need to state it as a dev dependency to get local development to work, both for Storybook but also for running the build process.